python/ray/experimental/channel/cpu_nccl_group.py

```diff
@@ -0,0 +1,273 @@
+import asyncio
+from collections import defaultdict
+from typing import Dict, List, Optional, Tuple
+from unittest import mock
+
+import torch
```

Comment:
We’re also noticing some build issues because this dependency isn’t found — is there another place we need to declare this file relies on torch?